example0.gz.sh

```diff
@@ -0,0 +1 @@
+./hashcat -t 32 -a 7 example0.hash ?a?a?a?a example.dict.gz
```

Comment:
Please rename back to example0.sh

Reply:
In the current version there're both example0.sh and example0.gz.sh
The only difference is that one uses the wordlist in plaintext and the other uses gz.
Do you want only the example0.sh with plaintext wordlist (original) or update example0.sh with new example.dict.gz ?

Reply:
Only the example0.sh with plaintext wordlist since the goal is to demonstrate a very easy use of hashcat

Reply:
ok, as soon as it works everything will remove both this file and the example wordlist in gzip